### DISCUSSION TITLE
Adding the label name to the screenshot

### DISCUSSION THREAD

**Discussion by agnanas** [biigle/core#955] [Ideas]
In image editing, the capture screenshot option is very useful especially if we want to discuss some doubtful ones or for educative purposes. While it shows the rectangles and circles drawn, would be great if it also overlays the label text on the image or somewhere on it. Can also be a configurable option to turn on and off in the downloaded screenshots.

And as a separate request, would be nice to also have this option in video editing when we can pause and just download a screenshot and with some label overlay as well.

**Comment by mzur** [comment]
Thanks for the idea! Adding the labels to the screenshot would be difficult because they are not drawn on the same surface than the image and outlines. But what do you think about this: There is already the option "Label Tooltip" in the settings. What if the option is extended to be either active on "hover" (current behavior), "always" (tooltips are shown for all annotations) or "off". So when you enable the tooltips "always" and create a regular screenshot (not using the button in BIIGLE) you should get what you describe, right? This way you can also take care of the correct positioning (so all tooltips are properly visible) before you do the screenshot.

The video screenshot request is now tracked here: https://github.com/biigle/core/issues/956

**Reply by agnanas** [reply]
Yeah that sounds like a good middle ground. Having all the labels visible at the same time can indeed enable us to take print screens or manual screenshots.

**Reply by mzur** [reply]
Great, this is now tracked here: https://github.com/biigle/core/issues/960
